auth.go

```diff
@@ -414,7 +414,8 @@ func (h *LogoutHandler) redirect(w http.ResponseWriter, r *http.Request, idToken
 	}
 
 	query := redirectURL.Query()
-	query.Set("post_logout_redirect_uri", r.Host)
+	// Use server config option instead of r.Host
+	query.Set("post_logout_redirect_uri", h.auth.conf.PostLogoutRedirectURL)
```

Comment:
Looking good, just a few changes are needed:

* Remove the redundant comment.
* If `PostLogoutRedirectURL` is not specified, the query param should default to host as before to not introduce a breaking change.